src/Lime.Transport.SignalR/EnvelopeHubOptions.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace Lime.Transport.SignalR
+{
+    internal class EnvelopeHubOptions
+    {
+        public int BackpressureLimit { get; set; }
```

Comment:
Use `BoundedCapacity` instead
```suggestion
        public int BoundedCapacity { get; set; }
```